src/main/scala/mesosphere/marathon/core/CoreModuleImpl.scala

```diff
@@ -199,10 +200,11 @@ class CoreModuleImpl @Inject() (
 
   // GROUP MANAGER
 
+  val groupManagerExecutionContext = NamedExecutionContext.fixedThreadPoolExecutionContext(8, "group-manager-module")
```

Comment:
Why 8 threads? Did we measure that 8 is a meaningful number? I'm not trying to overcomplicate things, but it seems to me that making these configurable would allow us to also easily run different tests to verify that the number makes sense.

Reply:
I did measurements with the different number of threads. Having more than 8 threads does not produce better numbers.